lms/djangoapps/grades/docs/troubleshooting/how-to-regrade-everyone-in-a-course.rst

```diff
@@ -0,0 +1,67 @@
+How to Update Grades for Every Learner in a Course
```

Comment:
@iloveagent57 Yay for investing in writing and adding this doc!
What do you think about moving it into a `how_tos` subfolder instead as recommended in [OEP-19#how-tos](https://open-edx-proposals.readthedocs.io/en/latest/oep-0019-bp-developer-documentation.html#how-tos)?

Reply:
👍